Autopilot/AttitudeManager/AttitudeManager.c

```diff
@@ -847,11 +850,11 @@ void readDatalink(void){
                 break;
             case ARM_VEHICLE:
                 if (*(int*)(&cmd->data) == 1234)
-                    startArm();
+                    armVehicle(500);
```

Comment:
armVehicle() is a function in either FixedWing.c or Multicopter.c. Both do different things.